src/uniparc/components/results/UniParcCard.tsx

```diff
@@ -76,13 +76,13 @@ const UniRefCard: FC<{
           </h5>
           <div className="result-card__info-container">
             <span className="result-card__info-bit">
-              <strong>Organism{organismCount === 1 ? '' : 's'}: </strong>
+              <strong>Organism{organismCount === 1 ? '' : 's'}:</strong>{' '}
               <LongNumber>{organismCount}</LongNumber>
             </span>
             <span className="result-card__info-bit">
-              <strong>UniprotKB entries:</strong> {uniProtKBCount.reviewed}{' '}
-              reviewed and <LongNumber>{uniProtKBCount.unreviewed}</LongNumber>{' '}
-              unreviewed
+              <strong>UniprotKB entries:</strong>{' '}
+              <LongNumber>{uniProtKBCount.reviewed}</LongNumber> reviewed and{' '}
```

Comment:
Something for later (was planning on doing it last sprint but ran out of time) but I think we should use the `Card` links prop for these (we'll have to do with all namespaces at the same time).